manifests/j/johnsadventures/JohnsBackgroundSwitcher/5.5.1.0.0/johnsadventures.JohnsBackgroundSwitcher.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: johnsadventures.JohnsBackgroundSwitcher
+PackageVersion: 5.5.1.0.0

```

```suggestion
PackageVersion: 5.5.1
```
This will require the version folder to be renamed